include/boost/graph/isomorphism.hpp

```diff
@@ -165,18 +163,23 @@ namespace detail
             BGL_FORALL_VERTICES_T(v, G1, Graph1)
             f[v] = graph_traits< Graph2 >::null_vertex();
 
+            std::size_t max_invariant;
```

Comment:
This variable is used uninitialized.  `-Wall` would catch this.  Suggest using ` = 0;` or something along those lines

Reply:
I've added the initialization.

I do think you're incorrect, though. `max_invariant` is never read from before being assigned a value, and adding `-Wall` does not show any `-Wmaybe_unitialized` notices.